### DISCUSSION TITLE
GAMMA in xticks, how to fix it to Greek alphabet style

### DISCUSSION THREAD

**Discussion by 1lifetime1** [SMTG-UCL/sumo#248] [Q&A] [answered]
![Uploading sumo_error.jpg…]()

**Comment by 1lifetime1** [comment]
![sumo_error](https://github.com/user-attachments/assets/d1340aad-54ee-4a6d-b7df-9a206423b6bc)

**Reply by 1lifetime1** [reply]
the figure was not well uploaded last time, now it is posted

**Comment by ajjackson** [comment, marked as answer]
You should get proper Γ characters if you used `sumo-kgen` to generate the KPOINTS file.

If `bandplot` is working from some other KPOINTS file it cannot know whether you intended "GAMMA" to be written literally. You should be able to get Γ by editing the file to replace "GAMMA" with "\Gamma". (We hope that nobody wants this to be written literally...)

**Reply by 1lifetime1** [reply]
Thanks so much, ajjackson. Following your instruction, I get the wanted figure, and problem is solved.
